deafrica-data/website/js/s3-bucket-listing.js

```diff
@@ -307,18 +308,18 @@ if (typeof AUTO_TITLE != 'undefined' && AUTO_TITLE == true) {
       } else {
         // CloudFront can't handle files bigger than 20 GB, so there's a special case needed
         if (parseInt(item.Bytes) > 20000000000) {
-          item.href = BUCKET_URL + '/' + encodeURIComponent(item.Key);
+          item.href = BUCKET_URL + '/' + item.Key;
         } else {
-          item.href = BUCKET_WEBSITE_URL + '/' + encodeURIComponent(item.Key);
+          item.href = BUCKET_WEBSITE_URL + '/' + item.Key;
         }
-        item.href = item.href.replace(/%2F/g, '/');
         //Converts to local user time
         var options = {}
         options.timeZone = 'UTC';
         options.timeZoneName = 'short';
         date = new Date(item.LastModified)
         item.LastModified = date.toLocaleString('en-GB', options)
       }
+      item.href = item.href.replace(/%2F/g, '/');
```

Comment:
what's this do?

Reply:
I see, because you also encode `/` in the Key name?

Reply:
Yeah, that's right.

This was there before. I'm just moving things around a bit and doing it for directories too.